assets/mdn-annotation.css

```diff
@@ -149,6 +149,10 @@
   background-image: url(https://cdnjs.loli.net/ajax/libs/browser-logos/41.0.0/android-webview-beta/android-webview-beta_32x32.png);
 }
 
+.mdn.wrapped div:nth-child(n + 3) {
```

Comment:
What does this actually do?

Reply:
Otherwise there will be blank space if multiple mdn slugs are associated with one box. It seems a bit ugly to me.
<img width="116" alt="图片" src="https://user-images.githubusercontent.com/20365185/56638936-f9e7da00-66a1-11e9-8a56-cb0434b495af.png">